inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/concat_4D.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/concat_4D.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+std::vector<std::vector<size_t>> inShapes = {
+    {1, 1, 33, 16},
+    {1, 1, 65, 16},
+};
+
+std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16};
+
+std::map<std::string, std::string> additional_config = {
+    {"GNA_COMPACT_MODE", "NO"},
+    {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
+    {"GNA_SCALE_FACTOR_0", "2000.0"},
```

Comment:
Not needed, the test framework calculates scale factor for all inputs

Reply:
This calculation does not factor in values in constant layers, which are higher than input values, since calculation is overvaluated by factor of 2 (1638 which allows range 0-20 when input is 0-10) it is all fine for this specific case, but for cases when non default value generators are used it is better to specify the scale factor.